src/main/webapp/js/edittask.js

```diff
@@ -0,0 +1,11 @@
+function validateForm() {
+    var text = document.forms["form"]["text"].value;
+    var date = document.forms["form"]["date"].value;
+    var status = document.forms["form"]["status"].value;
+    var priority = document.forms["form"]["priority"].value;
+    var tags = document.forms["form"]["tags"].value;
+    if ((status == "") || (priority == "") || (tags == "") || (text == "") || (text == "  ") || (date == "") ) {
+        alert("You haven't filled all fields!");
```

Comment:
Could you add fields which are not filled to the alert? I mean just to give a user an idea which field not filled.